src/pages/RepoPage/ATSTab/ATSNotConfigured.tsx

```diff
@@ -0,0 +1,40 @@
+import Button from 'ui/Button'
+
+function ATSNotConfigured() {
+  return (
+    <div className="flex flex-col justify-evenly space-y-7 bg-ds-gray-primary px-4 py-6">
+      <h1 className="mb-2 text-xl font-semibold">
```

Comment:
Are you familiar with the `gap` property? I find it nice to use instead of manual margins w/ mb-'s and stuff, also more widely used in our codebase. Maybe give that a try, it applies to all children if you put it in line 5 above as it works w/ flex. You should be able to remove all vertical styling with gap.